.github/workflows/prepare-release-pr.yml

```diff
@@ -11,6 +11,10 @@ on:
         description: 'Major release? (yes/no)'
         required: true
         default: 'no'
+      prerelease:
+        description: 'Prerelease (ex: "rc1", including the quotes)'
```

Comment:
Is the "including the quotes" still accurate now?

Reply:
Nope, thanks for catching it. 👍

Reply:
Fixed!